EIPS/eip-7701.md

```diff
@@ -49,17 +49,64 @@ Relying on the concept of "code sections" introduced with EIP-3540 appears to be
 
 ### Constants
 
-| Name       | Value |
-|------------|-------|
-| AA_TX_TYPE | TBD   |
+| Name              | Value               |
+|-------------------|---------------------|
+| AA_TX_TYPE        | TBD                 |
+| AA_ENTRY_POINT    | `address(0x7701)`   |
+| AA_SENDER_CREATOR | `address(ffff7701)` |
 
 ### New Transaction Type
 
 A new [EIP-2718](./eip-2718) transaction with type `AA_TX_TYPE` is introduced.
-Transactions of this type are referred to as
-"AA transactions".
+Transactions of this type are referred to as "AA transactions".
 
-The contents of such transactions are fully described in RIP-7560.
+
+Their payload should be interpreted as:
+
+```
+
+AA_TX_TYPE || rlp([
+  chainId,
+  nonce, 
+  sender, senderValidationData,
+  deployer, deployerData,
+  paymaster, paymasterData,
+  senderExecutionData,
+  builderFee,
+  maxPriorityFeePerGas, maxFeePerGas,
+  senderValidationGasLimit, paymasterValidationGasLimit,
+  senderExecutionGasLimit, paymasterPostOpGasLimit,
+  accessList,
+  authorizationList
+])
+
+```
+
+### Definitions
```

Comment:
Understanding the transaction flow from these definitions is hard.  We need a concise description of the transaction flow and a simple diagram. The definitions get into many details, which are needed but also drown the reader who is not familiar with the basic flow.

Maybe a pseudocode that looks like this (just more accurate) :
```
for tx in txs:
    if get_code(tx.sender) == 0:
        call_deploy(tx.deployer,tx.deployerData)
    assert call_section(tx.sender, role_sender_validation, tx)
    if tx.paymaster:
        assert call_section(tx.paymaster, role_paymaster_validation, tx)
    call_section(tx.sender, role_sender_execution, tx)
    if tx.paymaster has role_paymaster_post_tx:
        call_section(tx.paymaster, role_paymaster_post_tx, tx)
```